package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@segment/integration-intercom",
-  "version": "2.5.0",
+  "version": "2.6.0",
```

Comment:
Essentially I think making it clear that "hey something really big is changing" would be good here. It's very possible that we could accidentally rollback to a previous minor release without knowing the implications. We never do that for major versions as we know there's likely bigger changes that happened.